app/utils/validators.py

```diff
@@ -15,18 +15,18 @@ def validate_and_format_stock_symbol(stock_symbol: str) -> str:
 
     Parameters:
     -----------
-    stock_symbol: `str`
+    stock_symbol: ``str``
         stock symbol to be validated
             eg: "TCS", "INFY", etc.
 
     Raises:
     -------
-    HTTPException:
+    ``HTTPException``
         Raises when the stock symbol is not available in the Nse official website
 
     Return:
     -------
-    str:
+    str
```

Comment:
why didn't you add \`` to str type here?

Reply:
Check the docstrings in every file, Don't repeat these mistakes next time.

Reply:
added and checked all docstrings